src/js/facility-locator/reducers/searchQuery.js

```diff
@@ -0,0 +1,22 @@
+import { SEARCH_QUERY_UPDATED } from '../actions';
+// TODO (bshyong): flesh out shape of service Type object/options
+const INITIAL_STATE = {
+  searchString: '',
+  serviceType: null,
+  position: {
```

Comment:
I'm sure there are more, but here is a start of some similar decisions to make: https://github.com/department-of-veterans-affairs/vets-website/issues/3086